doc/parameters/param_list.md

```diff
@@ -480,6 +480,13 @@ When `planner_plugins` parameter is not overridden, the following default plugin
 | stop_on_failure | true | Whether to fail action task if a single waypoint fails. If false, will continue to next waypoint. |
 | loop_rate | 20 | Rate to check for results from current navigation task |
 
+## task_executor_at_waypoint plugin
```

Comment:
I think in the wp follower entry you need to add the `plugin` parameter that loads this one.